src/coinjoin/context.h

```diff
@@ -34,7 +34,7 @@ struct CJContext {
     std::shared_ptr<CJClientManager> clientman;
     std::shared_ptr<CCoinJoinClientQueueManager> queueman;
 #endif // ENABLE_WALLET
-    std::shared_ptr<CCoinJoinServer> server;
+    std::unique_ptr<CCoinJoinServer> server;
```

Comment:
seems as possible to add const here:
```
-    std::unique_ptr<CCoinJoinServer> server;
+    const std::unique_ptr<CCoinJoinServer> server;
```

Reply:
Adopted a more `LLMQContext`-like setup, resolved in latest push